aws/resource_aws_dx_lag.go

```diff
@@ -0,0 +1,162 @@
+package aws
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/directconnect"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsDxLag() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsDxLagCreate,
+		Read:   resourceAwsDxLagRead,
+		Update: resourceAwsDxLagUpdate,
+		Delete: resourceAwsDxLagDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"band_width": &schema.Schema{
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validateDxLagBandWidth,
+			},
+			"location": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"num_connections": &schema.Schema{
```

Comment:
Nitpick, but is there any reason to drift away from the name in the API? i.e. can't we just call it `number_of_connections `?